common/views/components/IIIFViewer/IIIFViewer.tsx

```diff
@@ -25,14 +31,55 @@ import NoScriptViewer from './parts/NoScriptViewer';
 import MainViewer from './parts/MainViewer';
 import ThumbsViewer from './parts/ThumbsViewer';
 import GridViewer from './parts/GridViewer';
+import ExplorePanel from './parts/ExplorePanel';
+import StructuresViewer from './parts/StructuresViewer';
 import Control from '../Buttons/Control/Control';
 import Layout12 from '@weco/common/views/components/Layout12/Layout12';
-
 import Download from '@weco/catalogue/components/Download/Download';
 import dynamic from 'next/dynamic';
 import { DigitalLocation, Work } from '../../../model/catalogue';
 import { FixedSizeList } from 'react-window';
 import useSkipInitialEffect from '@weco/common/hooks/useSkipInitialEffect';
+import BaseTabs, { TabType } from '../BaseTabs/BaseTabs';
```

Comment:
Could we put all of this 👇 behind the toggle so we aren't introducing code that isn't really tested  on prod?